akka-docs/rst/java/persistence.rst

```diff
@@ -177,32 +177,15 @@ They are cached and received by a persistent actor after recovery phase complete
 Recovery customization
 ^^^^^^^^^^^^^^^^^^^^^^
 
-Automated recovery on start can be disabled by overriding ``preStart`` with an empty or custom implementation.
+Automated recovery on start can be disabled by returning ``NoRecovery()`` in the ``recovery`` method of a ``PersistentActor``:
```

Comment:
If there is "Automated recovery" is there also some other kind of recovery?
Now, I don't think there is. Now there is always one recovery with the specified recovery parameter.